src/_tutorials/language/futures.md

```diff
@@ -11,22 +11,41 @@ nextpage:
 <div class="mini-toc" markdown="1">
   <h4>What's the point?</h4>
 
-  * Dart is single-threaded.
-  * Synchronous code can make your program freeze.
-  * Use `Future` objects (_futures_) to perform asynchronous operations.
-  * Use `await` in an async function to suspend execution until a future completes.
-  * Or use the `then()` method.
-  * Use try-catch expressions in async functions to catch errors.
-  * Or use the `catchError()` method.
-  * You can chain futures to run asynchronous functions in order.
+  * Dart code runs straight through, in a single "thread" of execution.
+  * Code that blocks the thread of execution can make your program freeze.
+  * `Future` objects (_futures_) represent asynchronous operations —
+    processing or I/O to be completed later.
+  * To suspend execution until a future completes,
+    use `await` in an async function (or use `then()`).
+  * To catch errors, use try-catch expressions in async functions
+    (or use `catchError()`).
+  * To run code concurrently, create an _isolate_
+    (or for web apps, a _worker_).
 </div>
 
-Dart is a single-threaded programming language. If any code blocks the thread
-of execution (for example, by performing a long-running calculation or
-blocking on I/O) the program effectively freezes. Asynchronous operations let
+Dart code runs in a predictable order, in a single "thread" of execution.
```

Comment:
Repeats the phrase used earlier, but with a different first half. Is that significant?

Reply:
I also oppose "predictable order" since it's ... not. Not really. We try to make futures complete in a *deterministic* order, but I wouldn't claim that the order is *predictable* to users. So, maybe, "deterministic".

Example:
```dart
var f1 = Future.value(42);
var f2 = f1.then((x) { print("f2"); return x + 1; });
var f3 = f1.then((x) { print("f3"); return x + 2; });
var f4 = f2.then((x) { print("f4"); return x + 3; });
```
I'm not sure how many will predict that this prints: "f2", "f4", "f3", in that order.